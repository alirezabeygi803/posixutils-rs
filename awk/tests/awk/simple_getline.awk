{
	print $0;
	print NF, NR, FNR;
	print(getline);
	print NF, NR, FNR;
	print $0;
}

{
	print NF, NR, FNR;
	print $0;
}