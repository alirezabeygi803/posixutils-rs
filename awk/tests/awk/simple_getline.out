1 Jane janitor 30
4 1 1
1
4 2 2
2 Smith sailor 45
4 2 2
2 Smith sailor 45
3 Roselyn radiologist 28
4 3 3
1
4 4 4
4 Megan machinist 32
4 4 4
4 Megan machinist 32
5 Celine clergy 87
4 5 5
1
4 6 6
6 Fredrick farmer 19
4 6 6
6 Fredrick farmer 19
7 Benjamin programmer 53
4 7 7
0
4 7 7
7 Benjamin programmer 53
4 7 7
7 Benjamin programmer 53
